tss-esapi/src/error/return_code/tpm/format_zero.rs

```diff
@@ -85,7 +85,7 @@ impl std::fmt::Display for TpmFormatZeroResponseCode {
         match self {
             TpmFormatZeroResponseCode::Error(e) => e.fmt(f),
             TpmFormatZeroResponseCode::Warning(e) => e.fmt(f),
-            TpmFormatZeroResponseCode::VendorSpecific(_) => write!(f, "Vendor specific error"),
+            TpmFormatZeroResponseCode::VendorSpecific(_) => write!(f, "Vendor specific error."),
```

Comment:
Why not print the vendor specific u16 in here too? (maybe formatting in hex?). When bad stuff happens I like to have as much info as possible :)

Reply:
```suggestion
            TpmFormatZeroResponseCode::VendorSpecific(code) => write!(f, "Vendor specific error: 0x{code:x}."),
```

(note: I didn't test it :sweat_smile: )

Reply:
Because usually all the the API calls looks like this:

```rust
        ReturnCode::ensure_success(
            unsafe {
                // Call to some ESYS API
            },
            |ret| {
                error!("Error when performing <some esys api>: {:#010X}", ret);
            },
        )?;
```

So I am not so sure the return code will be printed any way. I am not sure the return code without layer information will add anything to that.